package/index.ts

```diff
@@ -4572,6 +4576,28 @@ export function runTransition(transition: IRouteTransition): void {
     nextIteration();
 }
 
+interface IBobrilAnchorCtx { lastHandledTransitionRunCount: number; }
+
+export function anchor(node: IBobrilNode, name?: string, params?: Params): IBobrilNode {
+    postEnhance(node, {
```

Comment:
I will rewrite it to composition it will allow to change node to IBobrilChildren for free.